crates/sui-framework/packages/bridge/sources/message.move

```diff
@@ -253,6 +253,8 @@ module bridge::message {
         // bcs serialzies u64 as 8 bytes
         payload.append(reverse_bytes(bcs::to_bytes(&amount)));
 
+        assert!(vector::length(&payload) == 64, 0);
```

Comment:
we need to define an error code here and not use 0. If you look at the beginning of this file there are a group of `const E...` variables. Let's define a new one.
Also is the idea that the payload must be 64 bytes?

Reply:
Heard on the error message (there seem to be a lot of 0 error messages....) 

In Solidity we have: 

`require(_payload.length == 64, "BridgeUtils: TokenTransferPayload must be 64 bytes");`

The audit identified this asymmetry and I think its correct that we want this on both sides.